index.html

```diff
@@ -6,11 +6,13 @@
   </head>
   <body>
     <header>
-      <span class='brand'>Gmail Sender</span>
       <ul class='nav' role='nav'>
         <li><a href="http://www.scheduled.email/auth/google_oauth2/new?redirect_to=http://localhost:8080">Sign In</a>
         <li><a href="http://www.scheduled.email/sign-out?redirect_to=http://localhost:8080" class='new-window'>Sign Out</a>
         <li><a href="#new-schedule" class="new-schedule">+ Schedule</a></li>
+        <div class = "logo">
```

Comment:
this `div` should be within the `li`. Within a `ul` it's direct children should only be `li` elements, within those `li` elements you can have other stuff, like a `div`